tmt/plugins/__init__.py

```diff
@@ -6,6 +6,7 @@
 import os
 import pkgutil
 import sys
+from typing import Generator, List, Optional
```

Comment:
@ukulekek List is not needed afaict

```suggestion
from typing import Generator, Optional
```